{"uuid": "031e4555-cc1d-4f13-a207-e4d688e5c8ff", "children": ["54794d02-574c-4950-96fe-da9aa823dcd6"], "befores": [{"name": "browser", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchElementException: Message: Unable to find element with xpath == //button[.='Next']\n\n", "trace": "  File \"c:\\users\\patelvi\\appdata\\local\\programs\\python\\python37-32\\lib\\site-packages\\pluggy\\callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"c:\\users\\patelvi\\appdata\\local\\programs\\python\\python37-32\\lib\\site-packages\\_pytest\\fixtures.py\", line 949, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"c:\\users\\patelvi\\appdata\\local\\programs\\python\\python37-32\\lib\\site-packages\\_pytest\\fixtures.py\", line 775, in call_fixture_func\n    res = next(it)\n  File \"C:\\Users\\patelvi\\PycharmProjects\\NCE\\onx\\test_multi.py\", line 51, in browser\n    driver.find_element_by_xpath(\"//button[.='Next']\").click()\n  File \"c:\\users\\patelvi\\appdata\\local\\programs\\python\\python37-32\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 394, in find_element_by_xpath\n    return self.find_element(by=By.XPATH, value=xpath)\n  File \"c:\\users\\patelvi\\appdata\\local\\programs\\python\\python37-32\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 978, in find_element\n    'value': value})['value']\n  File \"c:\\users\\patelvi\\appdata\\local\\programs\\python\\python37-32\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 321, in execute\n    self.error_handler.check_response(response)\n  File \"c:\\users\\patelvi\\appdata\\local\\programs\\python\\python37-32\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 242, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1574168027961, "stop": 1574168073578}], "start": 1574168027961, "stop": 1574168074139}